src/connector/metrics.rs

```diff
@@ -14,8 +14,16 @@ where
     U: Future<Output = crate::Result<T>>,
 {
     let span = info_span!("quaint:query", "db.statement" = %query);
```

Comment:
In the query engine binary and node API, This span is [filtered-in by target](https://github.com/prisma/prisma-engines/blob/61863c7b1810c243ede42ae8ed40dc849b38c193/query-engine/query-engine-node-api/src/logger.rs#L35) , because it is user facing but doesn't have a `user_facing` attribute. Should we add the attribute to prevent this additional filtering based on the target? 


I know, it's coupling quiant to engine needs. But isn't that already based on all the tracing features added to it?